src/docs/.vuepress/config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   ga: "UA-321220-4",
   themeConfig: {
     repo: "GradleUp/shadow",
+    docsBranch: 'main',
```

Comment:
Can you send this change to a new PR?

Reply:
https://github.com/GradleUp/shadow/pull/1002